R/util.R

```diff
@@ -46,3 +46,14 @@ system_file <- function(path, package) {
 list_to_integer <- function(x) {
   vapply(x, identity, integer(1))
 }
+
+
+list_to_sexp <- function(x) {
+  if (is.recursive(x)) {
+    as.call(lapply(x, list_to_sexp))
```

Comment:
this means you can have multi-stage expressions like `1/2 * 3`?

Reply:
yeah, if odin supports that, it should work!

Reply:
```
sigma <- user(2 + 1/2 * 2)
```
this works fine. Recursion is a hell of a drug

Reply:
Yeah, I tried it, nice!